scalableminds/webknossos
# Limit bbox of volume tracings to that of fallback layer

When creating or uploading a non-task volume annotation layer with a fallback segmentation layer, the annotation layer’s bounding box will now be limited to that layer’s, instead of the whole dataset’s.

### URL of deployed dev instance (used for testing):
- https://volumefallbackbbox.webknossos.xyz

### Steps to test:
- On a datast with a segmentation layer that is smaller than the color layer,
- Create a volume annotation with fallback segmentation (is the default)
- bbox should be limited to fallback layer’s
- should still look the same after download + reupload
- Note that the frontend does not currently “respect” the volume tracing’s bbox. This is fixed in https://github.com/scalableminds/webknossos/pull/8602 – it might make sense to test both PRs together
 
### Issues:
- fixes #7365

- Related PRs #8535, #8551, #8602
- Related issues #5775, #6303, #8067


------
